salt/metalk8s/kubeadm/init/control-plane/lib.sls

```diff
@@ -1,7 +1,7 @@
 # Share macro for building image names for each control-plane component
-{% set repository = "localhost:5000/metalk8s-2.0" %}
-{% set kubernetes_version = "1.11.7" %}
+{%- set registry_url = "localhost:5000" -%}
```

Comment:
This is not really a URL :-P

Reply:
What would you call it? `registry_host` / `registry_address`?

Reply:
Used `registry_host`